resqpy/property.py

```diff
@@ -792,6 +792,15 @@ def parts(self):
 
       return list(self.dict.keys())
 
+   def uuids(self):
+      """Return list of uuids in this collection.
+
+      returns:
+         list of uuids being the members of this collection; there is one uuid per property array
+
+      :meta common:
+      """
+      return [self.model.uuid_for_part(p) for p in self.dict.keys()]
```

Comment:
Could alternatively make use of the uuid stored in the dictionary entry:
[self.uuid_for_part(p) for p in self.dict.keys()]
or
[values[2] for values in self.dict.values()]

Reply:
I don't follow your comment - isn't this already being done that way?